Source/MQTTnet.AspnetCore/ReaderExtensions.cs

```diff
@@ -60,15 +60,15 @@ private static bool TryReadBodyLength(ref ReadOnlySequence<byte> input, out int
             var index = 1;
             result = 0;
 
-            var temp = input.Slice(0, Math.Min(5, input.Length)).GetMemory();
+            var temp = input.Slice(0, Math.Min(5, input.Length)).GetMemory().Span;
```

Comment:
Why Slice input to create a Span<T> only to make Memory<T> out of it only to make Span<T> of it?  Can´t you just use input.Slice directly?

Reply:
i dont understand your point. 

I Slice the input directly, but that just gives me a shorter  ReadOnlySequence<byte> and that does not have an index operator so I need to get a span out of it

Reply:
my bad :)